Scripts/DCS-BIOS/lib/modules/aircraft_modules/M-2000C.lua

```diff
@@ -0,0 +1,905 @@
+module("M-2000C", package.seeall)
+
+local Module = require("Module")
+
+--- @class M_2000C: Module
+local M_2000C = Module:new("M-2000C", 0x7200, { "M-2000C" })
+
+--v1.39 by Ergo,Matchstick,MisterKnife,WarLord,Espresso29470
+
+--remove Arg# Pilot 1000
+--Get Displays Functions
+
+-- parse radios
+local vhfFrequency = ""
+local uhfFrequency = ""
+
+M_2000C:addExportHook(function()
+	local radios = M_2000C.parse_indication(7)
+	if not radios then
+		return
+	end
+
+	local vhf = coerce_nil_to_string(radios.text_COM_UHF1)
+	if vhf then
+		vhfFrequency = vhf:sub(1, 3) .. vhf:sub(5, 6)
+	end
+
+	local uhf = coerce_nil_to_string(radios.text_COM_UHF2)
+	if uhf then
+		uhfFrequency = uhf:sub(1, 3) .. uhf:sub(5, 6)
+	end
+end)
+
+-- parse fuel
+local fuelFlow = ""
+local fuelJauge = ""
+local fuelTotal = ""
+local FUEL_FLOW_LEN = 3
+
+M_2000C:addExportHook(function(dev0)
+	local fuel = M_2000C.parse_indication(3)
+	if not fuel then
+		return
+	end
+	fuelFlow = padLeft(fuel.txt_fuel_g, FUEL_FLOW_LEN)
+	local jauge_thous = BIOS.util.round(dev0:get_argument_value(349), 1, true)
```

Comment:
We should probably try to avoid using BIOS.util functions - let's use `Module.round` here instead (though, this seems to take 3 arguments? Why?)

Reply:
It's only M-2000C using that function so make it local then in the module. This function takes decimal places and whether to return number or string.